service/src/main/java/bio/terra/workspace/service/policy/TpsApiDispatch.java

```diff
@@ -0,0 +1,211 @@
+package bio.terra.workspace.service.policy;
+
+import bio.terra.common.logging.RequestIdFilter;
+import bio.terra.policy.api.TpsApi;
+import bio.terra.policy.client.ApiClient;
+import bio.terra.policy.client.ApiException;
+import bio.terra.policy.model.TpsComponent;
+import bio.terra.policy.model.TpsObjectType;
+import bio.terra.policy.model.TpsPaoCreateRequest;
+import bio.terra.policy.model.TpsPaoGetResult;
+import bio.terra.policy.model.TpsPaoReplaceRequest;
+import bio.terra.policy.model.TpsPaoSourceRequest;
+import bio.terra.policy.model.TpsPaoUpdateRequest;
+import bio.terra.policy.model.TpsPaoUpdateResult;
+import bio.terra.policy.model.TpsPolicyInputs;
+import bio.terra.policy.model.TpsUpdateMode;
+import bio.terra.workspace.app.configuration.external.FeatureConfiguration;
+import bio.terra.workspace.app.configuration.external.PolicyServiceConfiguration;
+import bio.terra.workspace.service.policy.exception.PolicyServiceAPIException;
+import bio.terra.workspace.service.policy.exception.PolicyServiceAuthorizationException;
+import bio.terra.workspace.service.policy.exception.PolicyServiceDuplicateException;
+import bio.terra.workspace.service.policy.exception.PolicyServiceNotFoundException;
+import io.opencensus.contrib.http.jaxrs.JaxrsClientExtractor;
+import io.opencensus.contrib.http.jaxrs.JaxrsClientFilter;
+import io.opencensus.contrib.spring.aop.Traced;
+import io.opencensus.trace.Tracing;
+import java.io.IOException;
+import java.util.Optional;
+import java.util.UUID;
+import javax.ws.rs.client.Client;
+import org.apache.commons.lang3.StringUtils;
+import org.slf4j.MDC;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.stereotype.Component;
+
+@Component
+public class TpsApiDispatch {
+  // TODO: PF-1714 pass through bearer token in all cases
```

Comment:
Is this still correct? Seems like we're adding the token in the client on line 64.

Reply:
Removed the comment.